app/models/energy/simulationparameter.py

```diff
@@ -0,0 +1,192 @@
+from pydantic import BaseModel, Schema, validator, ValidationError
+from typing import List, Union
+from enum import Enum
+from app.models.common.datetime import Date
+
+
+class ReportingFrequency(str, Enum):
+    timestep = 'Timestep'
+    hourly = 'Hourly'
+    daily = 'Daily'
+    monthly = 'Monthly'
+    annual = 'Annual'
+
+
+class SimulationOutput(BaseModel):
+    """Lists the desired outputs from the simulation and the format in which to report them."""
+
+    type: Enum('SimulationOutput', {'type': 'SimulationOutput'})
+
+    reporting_frequency: ReportingFrequency = ReportingFrequency.hourly
+
+    include_sqlite: bool = Schema(
+        default=True
+    )
+
+    include_html: bool = Schema(
+        default=False
+    )
+
+    outputs: List[str] = Schema(
+        default=None
+    )
+
+    summary_reports: List[str] = Schema(
+        default=None
+    )
+
+
+class SimulationControl(BaseModel):
+    """Used to specify which types of calculations to run."""
+
+    type: Enum('SimulationControl', {'type': 'SimulationControl'})
+
+    do_zone_sizing: bool = Schema(
+        default=True
+    )
+
+    do_system_sizing: bool = Schema(
+        default=True
+    )
+
+    do_plant_sizing: bool = Schema(
+        default=True
+    )
+
+    run_for_run_periods: bool = Schema(
+        default=True
+    )
+
+    run_for_sizing_periods: bool = Schema(
+        default=False
+    )
+
+
+class SolarDistribution(str, Enum):
+    minimal_shadowing = 'MinimalShadowing'
+    full_exterior = 'FullExterior'
+    full_interior_and_exterior = 'FullInteriorAndExterior'
+    full_exterior_with_reflection = 'FullExteriorWithReflections'
+    full_interior_and_exterior_with_reflections = 'FullInteriorAndExteriorWithReflections'
+
+
+class CalculationMethod(str, Enum):
+
+    average_over_days_in_frequency = 'AverageOverDaysInFrequency'
+    timestep_frequency = 'TimestepFrequency'
+
+
+class ShadowCalculation(BaseModel):
+    """Used to describe settings for EnergyPlus shadow calculation."""
+
+    type: Enum('ShadowCalculation', {'type': 'ShadowCalculation'})
+
+    solar_distribution: SolarDistribution = SolarDistribution.full_interior_and_exterior_with_reflections
+
+    calculation_frequency: int = Schema(
+        30,
+        ge=1
+    )
+
+    calculation_method: CalculationMethod = CalculationMethod.average_over_days_in_frequency
+
+    maximum_figures: int = Schema(
+        15000,
+        ge=200,
+        description='Number of allowable figures in shadow overlap calculations.'
+    )
+
+
+class SizingParameter(BaseModel):
+    """Allows the user to specify global heating and cooling sizing ratios."""
+
+    type: Enum('SizingParameter', {'type': 'SizingParameter'})
+
+    heating_factor: float = Schema(
+        1.25,
+        gt=0,
+        description='The global heating sizing ratio applied to all of the zone design heating loads and air flow rates.'
+    )
+
+    cooling_factor: float = Schema(
+        1.25,
```

Comment:
One other small comment.  The default values according to ASHRAE should be 1.25 for heating and 1.15 for cooling.  So I think this one here should be changed to 1.15.